test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs

```diff
@@ -1173,8 +1178,8 @@ public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSource_Warns()
                     packageX);
 
                 // Act
-                var result = _msbuildFixture.RunMsBuild(pathContext.WorkingDirectory, $"/t:restore {pathContext.SolutionRoot} /p:RestorePackagesConfig=true", ignoreExitCode: true);
-
+                string args = $"/t:restore {pathContext.SolutionRoot} /p:RestorePackagesConfig=true /p:RestoreUseStaticGraphEvaluation={useStaticGraphEvaluation}";
```

Comment:
Is this catching anything different from the other test? Feels redundant.

Reply:
To be honest, I just searched for packages.config tests and then made then run both with & without static graph restore. I didn't look at what the tests were actually doing.